apps/api/migrations/email-step-ui-schema-html-editor/email-step-ui-schema-html-editor-migration.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { expect } from 'chai';
+import { UserSession } from '@novu/testing';
+import { StepTypeEnum, UiComponentEnum, UiSchemaGroupEnum, WorkflowCreationSourceEnum } from '@novu/shared';
+import { Novu } from '@novu/api';
+import { run } from './email-step-ui-schema-html-editor-migration';
+
+describe('Update email step ui schema migration test #novu-v2', () => {
+  let session: UserSession;
+  let novuClient: Novu;
+  const workflows = ['test', 'test2'];
+
+  beforeEach(async () => {
+    session = new UserSession();
+    await session.initialize();
+
+    for (const workflow of workflows) {
+      await session.testAgent.post(`/v2/workflows`).send({
+        __source: WorkflowCreationSourceEnum.DASHBOARD,
+        name: workflow,
+        workflowId: workflow,
+        steps: [
+          {
+            name: 'email',
+            type: StepTypeEnum.EMAIL,
+            controlValues: {
+              body: '',
+              subject: '',
+            },
+          },
+        ],
+      });
+    }
```

Comment:
Ideally, we should use the `internal-sdk`, but I wasn't able to reuse it quickly in a reasonable amount of time. I was getting a 401 from somewhere when I used the API key.